nix/modules/lanzaboote.nix

```diff
@@ -13,6 +13,10 @@ let
     timeout ${toString timeout}
     console-mode ${config.boot.loader.systemd-boot.consoleMode}
   '';
+
+  # This is the fwupd-efi package. We need to get it this way because a user might override services.fwupd.package,
+  # which may cause pkgs.fwupd-efi to be a different package than what the fwupd package has as dependency.
+  fwupd-efi = builtins.head (builtins.filter (x: (x.pname or "") == "fwupd-efi") config.services.fwupd.package.buildInputs);
```

Comment:
Why not upstream that fwupd-efi is added to passthru?

Reply:
That would be the better idea, honestly. I'll make a PR to nixpkgs later to passthru it

Reply:
I've added a `TODO` for now to change it once a PR to add `passthru.fwupd-efi` to the `fwupd` package makes it to nixos-unstable. Thank you!

Reply:
Opened as NixOS/nixpkgs#221155

Reply:
Aaand merged

Reply:
I've pushed an update that will hopefully work once flake inputs are updated. Thank you both!